test/connection.test.ts

```diff
@@ -71,7 +71,7 @@ describe('Connection', function () {
     })
 
     it('should expose the minimumAcceptableExchangeRate', function () {
-      assert.equal(this.clientConn.minimumAcceptableExchangeRate, '0.5')
```

Comment:
Ideally you should add tests to cover this case, rather than modifying existing ones

Reply:
Got it. Edited the old tests (where relevant) to use the slippage of 0 explicitly so they pass, and added new ones as well.